channels-and-providers/sms/clickatell.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Clickatell'
+description: 'Learn how to use the Clickatell provider to send sms notifications using Novu'
+---
+
+You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform with a single API to create multi-channel experiences.
+
+# Getting Started
+
+To use the Clickatell provider in the SMS channel, the first step is to create a Clickatell account and add your API key to the Clickatell integration on the Novu platform.
```

Comment:
'To use Clickatell, you need to [create a Clickatell account](insert a link to the sign-up page of Clickatell) and get your API key. You'll need to plug this into Clickatell integration on [Novu web](insert link to the integration section of Novu web)